pkg/kube/template_test.go

```diff
@@ -77,536 +75,34 @@ func TestNewTemplate(t *testing.T) {
 			t.Fatalf("template does contain <placeholders> so GetSecrets should be called")
 		}
 	})
-}
-
-func TestToYAML_Deployment(t *testing.T) {
```

Comment:
Did you mean to delete all these tests?

Reply:
Ah shoot, was wondering why the coverage dropped so much